service/waterfall.go

```diff
@@ -375,10 +375,13 @@ func getVersionsAndVariants(skip, numVersionElements int, project *model.Project
 
 		failedAndStartedTasks := []task.Task{}
 		for _, tasks := range tasksByBuild {
-			for _, t := range tasks {
+			for i, t := range tasks {
 				if t.Status == evergreen.TaskFailed || t.Status == evergreen.TaskStarted {
-					if err = t.MergeNewTestResults(); err != nil {
-						return versionVariantData{}, errors.Wrap(err, "error merging test results")
+					if i < 200 { // Prevent cases where the waterfall will not load due to a large number of failed tasks.
```

Comment:
An alternative strategy that occurs to me is to let `t.MergeNewTestResults` take a limit on the number of test results that we will load. 
I think this would be better suited for the case at hand: the problem version doesn't have too many tasks, but rather the tasks have too many tests.

Reply:
Given that the waterfall will be deprecated within a quarter or so, and given that this would require counting test results, I would like to avoid this approach.